docs/maintenance/styleguide.md

```diff
@@ -49,9 +49,156 @@ Order buttons correctly:
 - ❌ Cancel (negative) | Save (positive)
 - ✅ Delete (negative) | ... (space) | Save (standard) | Publish (primary)
 
-### Captilization
+### Capitalization
 
 **Form labels**:
 
 - ✅ Resource type
 - ❌ Resource Type
+
+
+## Development practices
+
+### jQuery
+
+The most common usage of jQuery is when we apply JS to jinja (HTML) elements. 
+
+#### Selectors
+
+```html
+<div id="my-tab-menu" class="ui tab menu">
+  <div class="item active">Menu 1</div>
+  ...
+</div>
+```
+
+✅ DO
+
+```javascript
+$('#my-tab-menu').tab();
+```
+
+❌ DON'T
+```javascript
+$('.menu.tab').tab();
+```
+
+With the second approach you apply the function to all tab menu nodes on the page. What if there is a second menu? Or react element with the same class? None of your components/elements will behave as expected.  
+
+
+### Semantic UI
+
+#### Documentation
+
+Save your time and frustration, read the documentation (before you start with semantic UI)!
```

Comment:
Comment: Add a link to the documentation and perhaps some key chapters of the docs.